rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java

```diff
@@ -49,22 +53,23 @@
 
     private final JobKeyValueTransformer jobKeyValueTransformer;
 
-    public SubmitJobCommand(String... params) {
-
+    public SubmitJobCommand(String... params) throws CLIException {
+        if (params == null || params.length == 0) {
+            throw new CLIException(REASON_INVALID_ARGUMENTS, "Workflow file path is required");
+        }
         this.pathname = params[0];
         if (params.length > 1) {
             this.variables = params[1];
         }
         this.jobKeyValueTransformer = new JobKeyValueTransformer();
+
     }
 
     @Override
     public void execute(ApplicationContext currentContext) throws CLIException {
-        File jobFile = new File(pathname);
-        if (!jobFile.exists()) {
-            throw new CLIException(REASON_INVALID_ARGUMENTS, String.format("'%s' does not exist.", pathname));
-        }
         try {
+            validateFilePath(currentContext);
+            File jobFile = new File(pathname);
```

Comment:
I tried to do so, but the specific CLI exceptions are no longer captured, and only the generic one is raised.